### PR TITLE
feat: add bot-lable data #1089

### PR DESCRIPTION
**Pull request by bifenglin**
1. create `bot` folder and  the template
2. add some bot-label data

### CODE REVIEW

**Comment by frank-zsy** [comment]
@bifenglin You should check the existing label data, there are several errors in this PR.

- The extension of the file should be `yml` but not `yaml`, with the file you added, the file is not included in label data.
- The `data` field should not be simple array, you should add a sub filed `github_user`, just like [X-lab label data](https://github.com/X-lab2017/open-digger/blob/master/labeled_data/communities/xlab/index.yml#L11).
- In `github_user` field, the array element should be user id but not user login, you just parsed the ids in the issue, you should use the [output of the bot in the issue](https://github.com/X-lab2017/open-digger/issues/1089#issuecomment-1344019017).
- You should add a `Bot` type into the [label types](https://github.com/X-lab2017/open-digger/blob/master/src/label_data_utils.ts#L9), otherwise you will get an error in unit test.

**Comment by bifenglin** [comment]
Thanks for advising, I have edited what ur said. And are there any problems?

File: labeled_data/bot/index.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+name: GitHub Bot
+type: Bot
+label:
+  - github_user
+github_user:
```
Comment (frank-zsy) [inline labeled_data/bot/index.yml:5]: The format is not correct, this line is redundant, and following lines need a more `tab`

File: labeled_data/bot/index.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  - 105309205 # user:VSCodeTriageBot
+  - 96656532 # user:SnoozeThis
+  - 8517910 # user:LombiqBot
+  - 25180681 # user:renovate-bot
+  - 53108186 # user:codestar-github-bot-1
+  - 71103415 # user:rjpagano
+  - 74294145 # user:zhangsansana "
```
Comment (frank-zsy) [inline labeled_data/bot/index.yml:12]: tailing `"` should be removed.

File: labeled_data/bot/index.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+name: GitHub Bot
+type: Bot
+label:
```
Comment (frank-zsy) [inline labeled_data/bot/index.yml:3]: This should be `data` but not `label`, please check the existing data: https://github.com/X-lab2017/open-digger/blob/master/labeled_data/communities/xlab/index.yml

File: labeled_data/bot/index.yml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+name: GitHub Bot
+type: Bot
+data:
+  label:
+    - github_bot
```
Comment (frank-zsy) [inline labeled_data/bot/index.yml:5]: You should delete these 2 lines, there is no sub label called `github_bot`

Comment (bifenglin) [reply labeled_data/bot/index.yml]: So, do I needn't add a label? And the sub key uses `github_user` representing GitHub account is OK?

**Comment by xgdyp** [comment]
And I think, in this label, we can add more bots whose name not like `%bot%`.

**Comment by frank-zsy** [comment]
> And I think, in this label, we can add more bots whose name not like `%bot%`.

Yes, actually from OpenLeaderboard user ranking list, we can find lots of bot account which is not GitHub Apps on GitHub.

**Comment by bifenglin** [comment]
That's no problem. And I have figured out plenty of bot accounts, And I will push pr in the future. So, this pr is good? I have fixed the label.

**Comment by xgdyp** [comment]
I think it's no problem. @frank-zsy what do you think?
Otherwise, can you checkout bots in OpenLeaderboard first? I think this would improve OpenLeaderboard a lot！

**Review by xgdyp — APPROVED** [review]

**Pull request merged**